include/soci/mysql/soci-mysql.h

```diff
@@ -287,6 +287,23 @@ struct mysql_session_backend : details::session_backend
         return "SELECT table_name AS 'TABLE_NAME' FROM information_schema.tables WHERE table_schema = DATABASE()";
     }
 
+    std::string get_column_descriptions_query() const override
+    {
+        return "SELECT column_name as \"COLUMN_NAME\","
```

Comment:
Minor, but I think using raw string literals would make this much more readable, if only by removing the need to escape double quotes inside the string.

Reply:
I assume this is a general comment as they appear in a couple of places